data.sql

```diff
@@ -55,3 +55,38 @@ UPDATE animals SET owner_id = (CASE
                                 WHEN name IN ('Charmander', 'Squirtle', 'Blossom', 'Angemon', 'Boarmon') THEN 4
                                 ELSE 5
                               END);
+
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id) VALUES
+  (1, 1),
+  (3, 2),
+  (3, 1),
+  (4, 2);
+
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
+  (1, 1, '2020-05-24'),
+  (1, 3, '2020-07-22'),
+  (2, 4, '2021-02-02'),
+  (3, 2, '2020-01-05'),
+  (3, 3, '2021-05-04'),
+  (4, 2, '2020-03-08'),
+  (4, 2, '2020-05-14'),
```

Comment:
![image](https://github.com/ahmidou09/Vet-clinic-database/assets/93039359/95094028-84b9-4cc7-82d6-49cf415c8c97)

- [ ] It would be nice if you double-check all the insertions again because I found some mistakes in the dates and order of the queries 😨 Kindly double-check the order and dates of each query to make your database have correct data 👍 I have highlighted one mistake which you can see in the image above 👀